packages/component_library/pubspec.yaml

```diff
@@ -20,6 +20,14 @@ dependencies:
   toast: ^0.3.0
   logging: ^1.0.2
 
+  collection: ^1.17.1
+
+  # Internal packages:
+  io_library:
+    path: ../io_library
+  command:
+    path: ../command
```

Comment:
Where are these used?
I can't find anything in the component lib that uses this

Reply:
was redundant indeed